deploy/examples/create-external-cluster-resources.py

```diff
@@ -775,7 +775,7 @@ def create_rgw_admin_ops_user(self):
         cmd = ['radosgw-admin', 'user', 'create', '--uid', self.EXTERNAL_RGW_ADMIN_OPS_USER_NAME, '--display-name',
                'Rook RGW Admin Ops user', '--caps', 'buckets=*;users=*;usage=read;metadata=read;zone=read']
         if self._arg_parser.dry_run:
-            return self.dry_run("ceph " + "".joing(cmd))
+            return self.dry_run("ceph " + " ".join(cmd))
```

Comment:
We **really** need a python linter in the CI to avoid further issues like this. @subhamkrai could you please take care of this?

Reply:
Sure, I'll open the issue and assign that to myself.